airflow/providers/openlineage/plugins/macros.py

```diff
@@ -26,22 +26,44 @@
     from airflow.models import TaskInstance
 
 
+def lineage_job_namespace():
+    """
+    Macro function which returns Airflow OpenLineage namespace.
+
+    .. seealso::
+        For more information take a look at the guide:
+        :ref:`howto/macros:openlineage`
+    """
+    return conf.namespace()
+
+
+def lineage_job_name(task_instance: TaskInstance):
+    """
+    Macro function which returns Airflow task name in OpenLineage format (`<dag_id>.<task_id>`).
+
+    .. seealso::
+        For more information take a look at the guide:
+        :ref:`howto/macros:openlineage`
+    """
+    return get_job_name(task_instance)
+
+
 def lineage_run_id(task_instance: TaskInstance):
     """
-    Macro function which returns the generated run id for a given task.
+    Macro function which returns the generated runId (UUID) for a given task.
 
-    This can be used to forward the run id from a task to a child run so the job hierarchy is preserved.
+    This can be used to forward the runId from a task to a child run so the job hierarchy is preserved.
 
     .. seealso::
-        For more information on how to use this operator, take a look at the guide:
+        For more information take a look at the guide:
         :ref:`howto/macros:openlineage`
     """
     if TYPE_CHECKING:
         assert task_instance.task
```

Comment:
I think we don't need this anymore, since we stopped using `task_instance.task`

Reply:
Fixed